src/flight-search/dataUtils.js

```diff
@@ -46,8 +46,12 @@ module.exports = {
     }
 
     function hasAirportChangeAtStopover(legs) {
-      for (var i = 0; i < legs.length - 1; i++) {
-        if (legs[i].arrivalAirportCode !== legs[i + 1].departureAirportCode) return true;
+      for (var i = legs.length; i--;) {
+        var segments = legs[i].segments;
+        for(var j=segments.length-1; j > 0; j--) {
```

Comment:
Can we make `for loop` consistent?
Either use `for (var i = ***.length; i--;) {` or `for(var j=***.length-1; j > 0; j--) {`

Also might consider write a test and need to commit the build files as well